apps/Cargo.toml

```diff
@@ -64,9 +65,10 @@ abciplus = [
 
 [dependencies]
 namada = {path = "../shared", features = ["wasm-runtime", "ferveo-tpke", "rand", "secp256k1-sign-verify"]}
-arse-merkle-tree = {package = "sparse-merkle-tree", git = "https://github.com/heliaxdev/sparse-merkle-tree", branch = "bat/arse-merkle-tree", features = ["std", "borsh"]}
 ark-serialize = "0.3.0"
 ark-std = "0.3.0"
+# branch = "bat/arse-merkle-tree"
+arse-merkle-tree = {package = "sparse-merkle-tree", git = "https://github.com/heliaxdev/sparse-merkle-tree", rev = "04ad1eeb28901b57a7599bbe433b3822965dabe8", features = ["std", "borsh"]}
```

Comment:
alphabetization stronk

Reply:
yeah we should actually have this as a lint or something on our `Cargo.toml`s. Would reduce merge conflicts in the future